streaming/eslint.config.mjs

```diff
@@ -0,0 +1,38 @@
+// @ts-check
+
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+
+import { baseConfig } from '../eslint.config.mjs';
+
+export default tseslint.config([
+  baseConfig,
+  {
+    languageOptions: {
+      globals: globals.node,
+
+      ecmaVersion: 2021,
+      sourceType: 'module',
+    },
+
+    rules: {
+      'no-delete-var': 'off',
```

Comment:
Let's keep the comments on the rules that were in the previous config, at least `no-delete-var`?